ddl-scripts/create_tables_yugabyte.sql

```diff
@@ -17,12 +17,12 @@ CREATE TABLE IF NOT EXISTS event_journal(
   meta_ser_manifest VARCHAR(255),
   meta_payload BYTEA,
 
-  PRIMARY KEY((slice, entity_type) HASH, persistence_id, seq_nr ASC)
+  PRIMARY KEY(persistence_id HASH, seq_nr ASC)
 );
 
 -- `event_journal_slice_idx` is only needed if the slice based queries are used
-CREATE INDEX IF NOT EXISTS event_journal_slice_idx ON event_journal(entity_type ASC, slice ASC, db_timestamp ASC)
-  INCLUDE (seq_nr, deleted);
+CREATE INDEX IF NOT EXISTS event_journal_slice_idx ON event_journal(slice ASC, entity_type ASC, db_timestamp ASC)
```

Comment:
the order of the slice and entity_type columns shouldn't matter much, but there was already some inconsistency with the postrgres definition, and this makes it easier to pre-split into more tablets:
```
SPLIT AT VALUES ((31), (63), (95))
```

Reply:
Are we missing the `SPLIT` definiton? 

If I understand correctly, using split we instruct Yugabyte to distribute the data into the tablets according to the slice distribution we want. Is that correct? 

What will be the effect of it if we want to change the range we query (narrowing it)? I guess we will have more than one query hitting the same tablet. 
Would we get better performance if we re-create the index if we decide to change the range?

Reply:
Without the `SPLIT AT` definition it will use one tablet. That is probably fine as a default for an index table. At least we don't know for sure that it's a bottleneck yet so we should stick with that default.

Tablets can be split afterwards, with re-creating the index. There is also auto-splitting.